src/ucp/core/ucp_request.inl

```diff
@@ -819,7 +819,9 @@ ucp_recv_desc_init(ucp_worker_h worker, void *data, size_t length,
             return UCS_ERR_NO_MEMORY;
         }
 
-        padding = ucs_padding((uintptr_t)(rdesc + 1), alignment);
+        padding = ucs_padding((uintptr_t)
```

Comment:
use helper var to simplify
remove ( ) around rdesc+1
maybe can unite with ucp_am.c:1507 to have common macro/inline func to align with offset